Mathlib/Algebra/Order/GroupWithZero/Canonical.lean

```diff
@@ -371,24 +371,17 @@ protected abbrev orderedAddCommMonoid [OrderedAddCommMonoid α] (zero_le : ∀ a
   { WithZero.partialOrder, WithZero.addCommMonoid with
     add_le_add_left := @add_le_add_left _ _ _ (WithZero.addLeftMono zero_le).. }
 
--- This instance looks absurd: a monoid already has a zero
 /-- Adding a new zero to a canonically ordered additive monoid produces another one. -/
-instance canonicallyOrderedAddCommMonoid [CanonicallyOrderedAddCommMonoid α] :
-    CanonicallyOrderedAddCommMonoid (WithZero α) :=
-  { WithZero.orderBot,
-    WithZero.orderedAddCommMonoid _root_.zero_le,
-    WithZero.existsAddOfLE with
+instance canonicallyOrderedAdd [AddZeroClass α] [Preorder α] [CanonicallyOrderedAdd α] :
```

Comment:
```suggestion
instance instCanonicallyOrderedAdd [AddZeroClass α] [Preorder α] [CanonicallyOrderedAdd α] :
```
The old instance was incorrectly named, I guess.

Reply:
Let's do this in a follow-up PR.